_source/_posts/2024-04-30-express-universal-logout.md

```diff
@@ -602,7 +668,7 @@ if (!res.ok)
 }}     
 ```
 
-The onNewTask function will now look like this:
+The onNewTask function will now look like this with a change made only to the `onNewTAsk` function:
```

Comment:
You should re-evaluate this sentence. It seems repetitive. Also change casing for `onNewTAsk`